db/config.cc

```diff
@@ -1154,7 +1154,9 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , log_to_stdout(this, "log_to_stdout", value_status::Used)
     , log_to_syslog(this, "log_to_syslog", value_status::Used)
     , _extensions(std::move(exts))
-{}
+{
+    add_tombstone_gc_extension();
```

Comment:
Why do we need to call it inside db::config::config for the tombstone_gc extension only?

Reply:
`db::config` did not support any extension. But now we add tombstone gc extension for each table and so it needs to be supported (`boost/schema_loader_test.cc` uses bare `db::config`)

Reply:
ok.